### DISCUSSION TITLE
Autocompletion for Helix configuration

### DISCUSSION THREAD

**Discussion by quomat** [helix-editor/helix#5763] [Ideas]
It would be nice if Helix supported autocompleting config keys in its `config.toml` file.

vscode has this, it makes discoverability of different config options even better.
![obraz](https://user-images.githubusercontent.com/84818106/216019918-b79af003-5d26-4afd-9dd6-dca6a902c1de.png)

It would depend on toml LSP, so it wouldn't need to work out of the box. If LSP fails that's fine, it's not a critical feature. But I do not know how to put the configuration symbols into it.

There have been discussions about default config file - while that's a good point, I feel like people would still delete 90% of the options and leave them defaults.

How do you feel about it?

**Comment by pascalkuthe** [comment]
You would need to generate a schema for tablo. See https://github.com/helix-editor/helix/issues/3901.
